subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformIntegrationTest.groovy

```diff
@@ -934,16 +934,23 @@ $fileSizer
         fails "resolve"
 
         then:
-        failure.assertHasCause """Found multiple transforms that can produce a variant of project :lib for consumer attributes:
+        failure.assertHasCause """Found multiple transforms that can produce a variant of project :lib with requested attributes:
```

Comment:
Shall we add a test with a transformation chain as well?

Reply:
Not sure that's worth it, we would only be testing the way a transform chain reports its name. Which I assume is tested elsewhere ;-)